Snake/BusinessLayer/GameSnake.cs

```diff
@@ -7,27 +7,30 @@
 namespace BusinessLayer
 {
     public class GameSnake
-    {
+    {
         enum Direction {right, down, left, up};
         Direction directionFacing;
         private int snakeLength;
         private List<Point> currentPosition;
 
         //Method to create new snake
         public List<Point> createFirstSnake(int width, int height, int newLength)
-        {
-            this.directionFacing = Direction.right;
+        {
+            this.directionFacing = Direction.right;
             this.snakeLength = newLength;   
             //create head node (head of snake)
             Point head = new BusinessLayer.Point(width / 2, height / 2);
```

Comment:
will the values passed into Point from anywhere always be divided by 2? if so then that logic should be inside Point.